example-data.json

```diff
@@ -0,0 +1,53 @@
+// GET /{trailId}/photos
```

Comment:
I noted elsewhere. Break this file out to be actual (usable) examples of data from your endpoints.
No comments in .json

Reply:
Also, since you are following a spec, good to link it somewhere. (If not already in the readme)

Reply:
fixed... thanks!